### PR TITLE
example: updating janus example with --play-without-decoding option

### PR DESCRIPTION
**Pull request by rprata**
Follow up of #559.

### CODE REVIEW

**Comment by rprata** [comment]
@jlaine follow my tests:
- Configure a `janus-gateway` with default room (`1234`) with h264 codec permission.
- You could use this docker image to setup local environment: https://github.com/linagora/docker-janus-gateway.
- Start video room example.

Is not possible to test it in [janus example meetecho](https://janus-legacy.conf.meetecho.com) because only vp8 is permitted in video room example.

**Comment by jlaine** [comment]
I would expect us to need the --video-codec option too, as we discussed in #559. I would rather be explicit about the fact we are short-circuiting codec negotiation.

A note that Janus needs some configuration to allow h.264 would probably be a good idea too?

**Comment by rprata** [comment]
`--video-codec` is necessary for Janus? The example works without this (I believe that is configured by janus gateway video room in negotiation). I'll update this with a note about Janus and `video-room` configuration.

**Comment by rprata** [comment]
We can merge only this PR (I'll add a note in README) and in the future update janus example with necessary fixes.

File: examples/janus/README.rst [hunk 1]
```diff
@@ -32,3 +32,17 @@ If you want to play a media file instead of sending green video frames, run:
...
+
+   $ python webcam.py --play-from video.ts --play-without-decoding
+
+You can generate an example of such a file using:
+
+.. code-block:: console
+
+   $ ffmpeg -f lavfi -i testsrc=duration=20:size=640x480:rate=30 -pix_fmt yuv420p -codec:v libx264 -profile:v baseline -level 31 -f mpegts video.ts
+
+In this case, janus video room must be configured to allow this kind of codec (for example video/H264).
```
[the remark this answers is not in the document]
Comment (rprata) [reply examples/janus/README.rst]: Fixed.

[the remark this answers is not in the document]
Comment (jlaine) [reply examples/janus/README.rst]: Thanks!

**Pull request merged**